src/bot.js

```diff
@@ -45,7 +48,7 @@ client.on("messageCreate", async (message) => {
 
 	// respond to ask command
 	if ((message.content.startsWith('!askai') || message.content.startsWith('!ask'))) {
-		let question = message.content.startsWith('!askai') ? message.content.slice(6) : message.content.slice(4);
+		let question = message.content.startsWith('!!askai') ? message.content.slice(6) : message.content.slice(4);
```

Comment:
Why change the command here? ``!!`` is for moderators/admin.

`!` command should be for user of the bot.